tsl/test/expected/hypercore_create.out

```diff
@@ -220,16 +242,40 @@ ERROR:  hypertable "test3" is missing compression settings
 \set ON_ERROR_STOP 1
 -- Add compression settings
 alter table test3 set (timescaledb.compress, timescaledb.compress_orderby='time desc', timescaledb.compress_segmentby='');
+select compress_reloptions from test_chunk_info where chunk = :'chunk'::regclass;
+ compress_reloptions 
+---------------------
+ 
+(1 row)
+
 alter table :chunk set access method hypercore;
+select compress_reloptions from test_chunk_info where chunk = :'chunk'::regclass;
```

Comment:
Is this really right?  I think autovacuum should be enabled on the "public" hypercore chunk but disabled on the "internal" compressed chunk, right?

Given the alter statement above, it looks like this is the "public" chunk. Why should we turn off autovacuum on that one?

Ideally, I'd like to see reloptions both for the "main" Hypercore chunk and its internal compressed auxiliary chunk. The main one should have autovacuum enabled while the internal one should not.

Reply:
No, is a view over all chunks and shows the reloptions for the "public" chunk as `chunk_reloptions` and reloptions for the compressed relation as `compress_reloptions`.

Since there seems to be some confusions around this, I just replaced the `compress_reloptions` with `*` to make it clear that we see reloptions for both the (non-compressed) chunk and the compressed chunk.